### ISSUE TITLE
where are all 1,280 images from? Imagenet training set or test set?

### ISSUE THREAD

**Issue by xxyh1993** [rgeirhos/texture-vs-shape#31]
Looking forward to your reply

**Comment by rgeirhos** [comment]
Thanks for your interest. They're not sourced from ImageNet. Kindly see Appendix A.6 of https://arxiv.org/pdf/1811.12231 for details.
